lib/qx/tool/cli/commands/Compile.js

```diff
@@ -165,6 +165,11 @@ qx.Class.define("qx.tool.cli.commands.Compile", {
             describe: "Whether bundling is enabled",
             type: "boolean",
             default: true
+          },
+          "force": {
+            describe: "Override warnings",
```

Comment:
Does it really make sense to create a `force*` flag for whatever specific warning we want to override? I'd rather use the know Linux-y way to have a generic  "force" CLI option that can be used in cases where the action is aborted for a specific reason. It will be rare that two separate warning will have to be output simultaneously and then will have to be overridden separately. I think the current naming is fine if we agree to use the generic `--force` only in "fatal" warnings that abort an action.